src/pages/dashboard/_CDCComparisonContainer.js

```diff
@@ -1,64 +1,101 @@
-import { sum } from 'd3-array'
-import React, { useMemo } from 'react'
+import { max, sum } from 'd3-array'
+import { timeParse } from 'd3-time-format'
+import React from 'react'
 import { graphql, useStaticQuery } from 'gatsby'
 
 import BarChart from './charts/_BarChart'
-import { calculateTotal, formatNumber, parseDate } from './_utils'
+import { formatNumber, parseDate } from './_utils'
+
+const parseCdcDate = timeParse('%m/%d/%Y')
+
+const sortChronologically = (a, b) => {
+  if (a.date > b.date) return 1
+  if (a.date < b.date) return -1
+  return 0
+}
 
 export default function CDCComparisonContainer() {
   const query = useStaticQuery(graphql`
     {
       allCovidUsDaily {
         nodes {
           date
-          positive
-          negative
+          totalTestResultsIncrease
+        }
+      }
+      allCdcDaily {
+        nodes {
+          dateCollected
+          lag
+          cdcLabs
+          dailyTotal
         }
       }
     }
   `)
-  const data = useMemo(() => {
-    const nodes = query.allCovidUsDaily.nodes
-      .map(node => [
-        {
-          date: parseDate(node.date),
-          value: calculateTotal(node),
-        },
-      ])
-      .flat()
-
-    return nodes.sort((a, b) => {
-      if (a.date > b.date) return 1
-      if (a.date < b.date) return -1
-      return 0
+  const cdcData = query.allCdcDaily.nodes
+    .map(node => {
+      const date = parseCdcDate(`${node.dateCollected}/2020`)
+      return {
+        date,
+        value: +node.dailyTotal,
+      }
     })
-  }, [query.allCovidUsDaily.nodes.length])
+    .sort(sortChronologically)
+  const data = query.allCovidUsDaily.nodes
+    .map(node => [
+      {
+        date: parseDate(node.date),
+        value: node.totalTestResultsIncrease,
+      },
+    ])
+    .flat()
+    .sort(sortChronologically)
+  const cdcCumulativeTotal = sum(cdcData, d => d.value)
```

Comment:
My understanding of the CDC data is that its all daily and none of it is cumulative, so we should be able to create a sum like this